### PR TITLE
JIRA ID :PD-669

### PR DESCRIPTION
**Pull request by npaulq2ebankingCK**
DESCRIPTION :Pr for BAO data
Test : NO
Comments : NO

### CODE REVIEW

**Review by jitendra30 — APPROVED** [review]

File: BAO/Configurations/genesis__Policy_Condition_Category_Association__c/Article_of_Organization_PolicyArticles_of_Organization.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+     "attributes": {
+          "type": "genesis__Policy_Condition_Category_Association__c"
+     },
+     "Name": "Article of Organization Document Policy",
+     "genesis__External_Id__c": "Article_of_Organization_PolicyArticles_of_Organization",
```
Comment (nirajsinhaCLS) [inline BAO/Configurations/genesis__Policy_Condition_Category_Association__c/Article_of_Organization_PolicyArticles_of_Organization.json:6]: Duplicate names are appended.

Comment (nirajsinhaCLS) [reply BAO/Configurations/genesis__Policy_Condition_Category_Association__c/Article_of_Organization_PolicyArticles_of_Organization.json:6]: Add underscore between two names.

Comment (nirajsinhaCLS) [reply BAO/Configurations/genesis__Policy_Condition_Category_Association__c/Article_of_Organization_PolicyArticles_of_Organization.json:6]: Please do for all Policy_Condition_Category_Association records.

File: BAO/Configurations/genesis__Policy_Exception_Definition__c/PED_0000009.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+     "attributes": {
+          "type": "genesis__Policy_Exception_Definition__c"
+     },
+     "genesis__Active__c": true,
+     "genesis__Condition_Type__c": "Document",
+     "genesis__Description__c": "Front Side of Driver's License/ID Policy",
+     "genesis__Enable_Override__c": true,
+     "genesis__Evaluation_Expression__c": "1",
+     "genesis__External_ID__c": "PED_0000009",
```
Comment (nirajsinhaCLS) [inline BAO/Configurations/genesis__Policy_Exception_Definition__c/PED_0000009.json:10]: Use genesis__Policy_Condition_Name__c field for the external id.

**Review by nirajsinhaCLS — CHANGES_REQUESTED** [review]

File: BAO/Configurations/genesis__Policy_Exception_Definition__c/PED_0000009.json (hunk 2 above)
Comment (nirajsinhaCLS) [reply BAO/Configurations/genesis__Policy_Exception_Definition__c/PED_0000009.json:10]: Please update this.

**Review by nirajsinhaCLS — CHANGES_REQUESTED** [review]

**Review by jitendra30 — APPROVED** [review]

**Review by nirajsinhaCLS — APPROVED** [review]

**Review by YashasGCCLS — APPROVED** [review]

**Pull request merged**